packages/mock-server/src/api/mutations/invoice.ts

```diff
@@ -68,8 +68,8 @@ export const InvoiceMutation = {
       const resolvedInvoice = ResolverService.invoice.byId(String(invoiceId));
 
       if (
-        resolvedInvoice.status === InvoiceNodeStatus.Confirmed ||
-        resolvedInvoice.status === InvoiceNodeStatus.Finalised
+        resolvedInvoice.status !== InvoiceNodeStatus.Allocated &&
+        resolvedInvoice.status !== InvoiceNodeStatus.New
       ) {
         throw new Error("Can't delete delivered or finalised invoice");
```

Comment:
or a picked, shipped or verified one?
Perhaps better to say `Only allocated and new shipments can be deleted` ?

Reply:
done!